widgetssdk/src/main/java/com/glia/widgets/core/callvisualizer/domain/CallVisualizer.java

```diff
@@ -0,0 +1,22 @@
+package com.glia.widgets.core.callvisualizer.domain;
+
+import android.content.Context;
+
+import com.glia.widgets.view.VisitorCodeView;
+
+/**
+ * Provides controls related to Call Visualizer module
+ * TODO: improve docs in MOB-1816
+ */
+public interface CallVisualizer {
+    /**
+     * This builds a VisitorCodeView component that can be integrated in the client application
+     */
+    VisitorCodeView createVisitorCodeView(Context context);
+
+    /**
+     * This shows Visitor Code inside a Dialog on top of current Activity inside of your application
+     */
+    void showVisitorCode(Context context);
```

Comment:
I would add Dialog to the end of the name, just to be clear what it does

Reply:
Fixed: 345ce4c